src/main/webapp/WEB-INF/tags/instructor/studentList/ajaxResult.tag

```diff
@@ -0,0 +1,173 @@
+<%@ tag description="instructorStudentList - Ajax result" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="courseId" required="true" %>
+<%@ attribute name="courseIndex" required="true" %>
+<%@ attribute name="hasSection" required="true" %>
+<%@ attribute name="sections" type="java.util.List" required="true" %>
+<table class="table table-responsive table-striped table-bordered margin-0">
+    <c:choose>
+        <c:when test="${not empty sections}">
+            <thead class="background-color-medium-gray text-color-gray font-weight-normal">
+                <tr id="resultsHeader-${courseIndex}">
+                    <th>Photo</th>
+                    <th id="button_sortsection-${courseIndex}" class="button-sort-none<c:if test="${not hasSection}"> hidden</c:if>" onclick="toggleSort(this,2)">
+                        Section <span class="icon-sort unsorted"></span>
+                    </th>
+                    <th id="button_sortteam-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,3)">
+                        Team <span class="icon-sort unsorted"></span>
+                    </th>
+                    <th id="button_sortstudentname-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,4)">
+                        Student Name <span class="icon-sort unsorted"></span>
+                    </th>
+                    <th id="button_sortemail-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,5)">
+                        Email <span class="icon-sort unsorted"></span>
+                    </th>
+                    <th>Action(s)</th>
+                </tr>
+                <tr id="searchNoResults-${courseIndex}" class="hidden">
+                    <th class="align-center color_white bold">Cannot find students in this course</th>
+                </tr>
+            </thead>
+            <tbody>
+                <c:set var="teamIndex" value="${-1}" />
+                <c:set var="studentIndex" value="${-1}" />
+                <c:forEach items="${sections}" var="section" varStatus="sectionIdx">
+                    <c:set var="sectionIndex" value="${sectionIdx.index}" />
+                    <%-- generated here but to be appended to #sectionChoices in instructorStudentList.jsp
+                         will be transported via JavaScript in instructorStudentListAjax.js --%>
```

Comment:
Should we reorganise these (the checkboxes) since we're breaking things into tag files?
Or maybe as a separate issue?
It feels odd having these between the table's header and contents.